server/db/recreate-schema.sql

```diff
@@ -1,10 +1,44 @@
 -- Drop tables in case they already exist
+
+DROP TABLE if exists work_experience;
+DROP TABLE if exists education;
+DROP TABLE if exists profiles;
 DROP TABLE if exists users;
 
+
 -- Create tables
 CREATE TABLE users (
   id SERIAL PRIMARY KEY,
   email    VARCHAR(200) NOT NULL,
   role     VARCHAR(15) NOT NULL,
   password VARCHAR(200) NOT NULL
 );
+
+CREATE TABLE profiles (
+  id  SERIAL PRIMARY KEY,
+  profile_public_id  INT REFERENCES users(id),
```

Comment:
As far as i understand, profile_public_id is not the user id. It should be a varchar which value will be generated in the API code. This value could be used later to refer to a profile and could be used to build the profile url for example.